polygeist/lib/Dialect/Polygeist/Transforms/LICM.cpp

```diff
@@ -762,12 +767,9 @@ class SYCLAccessorVersionBuilder : public SCFLoopVersionBuilder {
 public:
   SYCLAccessorVersionBuilder(
       LoopLikeOpInterface loop,
-      const SmallVectorImpl<AccessorPairType> &requireNoOverlapAccessorPairs)
+      const SmallVector<AccessorPairType> &requireNoOverlapAccessorPairs)
```

Comment:
The LLVM Programmers' Manual discourages use of `SmallVector` as parameter type. Any particular reason to use it here?

Reply:
The change was to match the member type. Now changed to `ArrayRef`.